.github/workflows/selfdrive_tests.yaml

```diff
@@ -305,6 +305,8 @@ jobs:
       with:
         submodules: true
     - uses: ./.github/workflows/setup
+    - name: Build Base Docker image (temp for ci)
+      run: eval "$BUILD"
```

Comment:
this shouldn't be temporary. can you make  separate PR to do this after this one is merged?